<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/zork.ico</normaloff>:/icon/zork.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="min">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/min.png</normaloff>:/images/min.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="max">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/max.jpg</normaloff>:/images/max.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>250</width>
      <height>160</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/logo.png</pixmap>
    </property>
   </widget>
   <widget class="CustButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/play1.png</normaloff>:/buttons/play1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="CustButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/help1.png</normaloff>:/buttons/help1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="CustButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="CustButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="buttons.qrc">
      <normaloff>:/buttons/setting.png</normaloff>:/buttons/setting.png</iconset>
    </property>
   </widget>
   <widget class="LoadingWidget" name="loadingWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>800</width>
      <height>540</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>240</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>480</width>
        <height>30</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="TextTicker" name="label2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>800</width>
        <height>220</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gabriola</family>
        <pointsize>36</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-color: rgb(255, 255, 127);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>390</y>
        <width>230</width>
        <height>60</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>CommandLinkButton</string>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="textLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>800</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Eurostile</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>250</width>
        <height>250</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="Game" name="game">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>780</width>
        <height>520</height>
       </rect>
      </property>
      <property name="sceneRect">
       <rectf>
        <x>0.000000000000000</x>
        <y>0.000000000000000</y>
        <width>0.000000000000000</width>
        <height>0.000000000000000</height>
       </rectf>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>300</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Felix Titling</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{background-color: qlineargradient(spread:pad, 
x1:0.5, y1:0, x2:0.5, y2:1, stop:0 
rgba(214,249,255,100), stop:1
rgba(158,232,250,100));
border: 2px solid rgba(205, 92, 92, 255);
border-radius:20px;
color: red;}</string>
      </property>
      <property name="plainText">
       <string>plainTextEdit
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="northButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#northButton{
background-color:rgba(100,225,100,30);
border-style:outset;
border-width:4px;
border-radius:10px;
border-color:rgba(255,255,255,30);
font:bold 20px;
color:red;
padding:6px;
}

QPushButton#northButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}

QPushButton#northButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}
</string>
      </property>
      <property name="text">
       <string>NORTH</string>
      </property>
     </widget>
     <widget class="QPushButton" name="westButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#westButton{
background-color:rgba(100,225,100,30);
border-style:outset;
border-width:4px;
border-radius:10px;
border-color:rgba(255,255,255,30);
font:bold 20px;
color:red;
padding:6px;
}

QPushButton#westButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}

QPushButton#westButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}
</string>
      </property>
      <property name="text">
       <string>WEST</string>
      </property>
     </widget>
     <widget class="QPushButton" name="eastButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>240</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#eastButton{
background-color:rgba(100,225,100,30);
border-style:outset;
border-width:4px;
border-radius:10px;
border-color:rgba(255,255,255,30);
font:bold 20px;
color:red;
padding:6px;
}

QPushButton#eastButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}

QPushButton#eastButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}
</string>
      </property>
      <property name="text">
       <string>EAST</string>
      </property>
     </widget>
     <widget class="QPushButton" name="southButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>450</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#southButton{
background-color:rgba(100,225,100,30);
border-style:outset;
border-width:4px;
border-radius:10px;
border-color:rgba(255,255,255,30);
font:bold 20px;
color:red;
padding:6px;
}

QPushButton#southButton:pressed{
background-color:rgba(100,255,100,200);
border-color:rgba(255,255,255,30);
border-style:inset;
color:rgba(0,0,0,100);
}

QPushButton#southButton:hover{
background-color:rgba(100,255,100,100);
border-color:rgba(255,255,255,200);
color:rgba(0,0,0,200);
}
</string>
      </property>
      <property name="text">
       <string>SOUTH</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="textEdit2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>430</y>
        <width>250</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">*{background-color: qlineargradient(spread:pad, 
x1:0.5, y1:0, x2:0.5, y2:1, stop:0 
rgba(214,249,255,100), stop:1
rgba(158,232,250,100));
border: 2px solid rgba(205, 92, 92, 255);
border-radius:20px;}</string>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="MyLabel" name="showImage1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="MyLabel" name="showImage2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>151</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustButton</class>
   <extends>QPushButton</extends>
   <header location="global">custbutton.h</header>
  </customwidget>
  <customwidget>
   <class>LoadingWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">loadingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextTicker</class>
   <extends>QLabel</extends>
   <header location="global">textticker.h</header>
  </customwidget>
  <customwidget>
   <class>Game</class>
   <extends>QGraphicsView</extends>
   <header location="global">game.h</header>
  </customwidget>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
  <include location="images.qrc"/>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
